pkg/systemd/services/service-details.jsx

```diff
@@ -224,6 +253,29 @@ export class ServiceDetails extends React.Component {
         return true;
     }
 
+    doPropertyPolling() {
+        systemd_client.call(this.props.unit.path,
+                            "org.freedesktop.DBus.Properties", "GetAll",
+                            [cockpit.format("org.freedesktop.systemd1.$0", this.unitTypeCapitalized)])
+                .then(result => {
+                    this.addUnitProperties({
+                        MemoryCurrent: (result[0].MemoryCurrent && result[0].MemoryCurrent.v > 0) ? result[0].MemoryCurrent.v : null,
+                        Listen: result[0].Listen ? result[0].Listen.v : null
```

Comment:
It makes sense that `MemoryCurrent` does not generate PropertiesChanged signals, but I'm a bit surprised that changes to `Listen` don't? Is it even realistic that `Listen` changes *while* a service is active? If we only need to poll for MemoryCurrent, then GetAll could be reduced to a much cheaper direct Get.

Reply:
In other words, the Listen property is defined by the unit's `ListenStream=` definition. So changing that would require a daemon-reload and a service restart.

Reply:
So I can remove the event listener I have, but in that case, I still need to call the Get API for Listen in componentDidUpdate or else move the fetching of Listen in the main services.jsx file, which actually refetches some unit properties on daemon reload or unit events. I would rather do the later.